docs/reference/localization.md

```diff
@@ -2,193 +2,256 @@
+
+Where possible in packages that use Fluent, we should instead use [Fluent's DATETIME function][#fluent-datetime-function].
 
 ## Terms of Service and Privacy Policy are handled differently
 The FxA [Terms of Service][#fxa-tos-source] and [Privacy Policy][#fxa-pp-source] are handled differently to other FxA translations. The most important difference is that these are considered legally binding documents, as such changes are driven by Mozilla’s legal team and source documents are held in the [mozilla/legal-docs repository][#legal-docs-repo] on Github. Any time either document is updated, paid specialist legal translators make the updates for other languages.
 
 Unlike the rest of the FxA translations that are held in gettext .po files, the terms of service and privacy policy source documents are full markdown documents to allow the Legal team control over the formatting of the documents. During the FxA docker image build, the head of the legal-docs repo is pulled in and the markdown documents converted to HTML.
 
-## Future directions / Possible improvements
-* Send a slack notification or [email][#email-on-extraction] whenever strings are extracted.
-* Notify a mailing list if l10n-lint fails.
-* Back out a commit if l10n-lint fails.
-* Extract strings more frequently, even up to every commit?
-  * Would the l10n team be on board with this? A stated goal was to minimize the number of pings a translator received for any given project to avoid notification burnout.
-* Strings are only bundled in with the production JS whenever docker images are created. During a normal release, a train is cut and a docker image created on a Monday. The train is then deployed the following Monday. If no new docker images are created in between, then an entire week of translations may not be deployed to production.
-* Automate checking Pontoon for newly translated locales. If a locale reaches a certain threshold (we have used 70% in the past), open a PR against the [supportedLanguages list][#supported-languages].
-
-[#add-sri-to-html]: https://github.com/mozilla/fxa/blob/dfc7a258402840e35139fd918c7bc93bc39ca9b7/packages/fxa-content-server/grunttasks/sriify.js#L74
-[#add-sri-to-js]: https://github.com/mozilla/fxa/blob/dfc7a258402840e35139fd918c7bc93bc39ca9b7/packages/fxa-content-server/grunttasks/sriify.js#L96
-[#devices-and-apps-panel]: https://accounts.firefox.com/settings/clients
-[#dist-template-dir]: https://github.com/mozilla/fxa/tree/main/packages/fxa-content-server/server/templates/pages
-[#docker-l10n-clone]: https://github.com/mozilla/fxa/blob/dfc7a258402840e35139fd918c7bc93bc39ca9b7/packages/fxa-content-server/Dockerfile-build#L19
+<!-- TODO review handling of legal docs in content-server vs react -->
```

Comment:
Is there a follow up for this? We could probably use a 1 pointer in the l10n improvements epic to review and add anything missing. I don't think we need content-server vs React here but we did have https://mozilla-hub.atlassian.net/browse/FXA-8040 filed, though it's been canceled.

Reply:
Filed a follow-up and removed the TODO from the docs: https://mozilla-hub.atlassian.net/browse/FXA-8774